.github/workflows/ci.yml

```diff
@@ -27,10 +29,15 @@ jobs:
       -
         name: Docker meta
         id: docker_meta
-        uses: crazy-max/ghaction-docker-meta@v1
+        uses: crazy-max/ghaction-docker-meta@v3
```

Comment:
Have the action moved to docker/metadata-action@v3?

Reply:
the semver pattern syntax didn't work with v1, so I upgraded it.

Reply:
Sorry, meant maybe change "crazy-max/ghaction-docker-meta" -> "docker/metadata-action"?

Reply:
aaah now I understand. Yes, you're right